inputfiles/overridingTypes.json

```diff
@@ -1073,5 +1073,23 @@
         "signatures": [
             "new(data?: string): Text"
         ]
+    },
+    {
```

Comment:
these methods are acctually defined in `inputFiles/addedTypes.json` see https://github.com/noobiept/TSJS-lib-generator/blob/3855f80a4f24e558633d620bfb22f5b67495af97/inputfiles/addedTypes.json#L1225, so i would move these there instead.

You should also see the change in dom.generated.d.ts above.